client/src/lib/utils/networkData.ts

```diff
@@ -1,48 +1,65 @@
 import { base } from "$app/paths";
-import { PUBLIC_FAUCET_ROCOCO_URL, PUBLIC_FAUCET_WESTEND_URL } from "$env/static/public";
+import { PUBLIC_FAUCET_URL } from "$env/static/public";
 
 export interface ChainData {
   name: string;
   id: number;
 }
 
+function faucetUrl(defaultUrl: string): string {
```

Comment:
@mutantcornholio sorry, i don't understand this swap. 
Why can't it leave in PUBLIC_FAUCET_\<network\>_URL and locally .env you make it as localhost:5555 or prod url? 
i think "unsetting" PUBLIC_FAUCET_URL to "" in order to make calls to prod, vs setting prod URL explicitly to PUBLIC_FAUCET_WESTEND_URL would be more clear and expected?

Reply:
My reasoning is, to reduce the amount of necessary variables, with the growth of faucet deployments.   

These URLs are static, and AFAIK the only valid reason to override them, is to route faucet client to dev faucet backend.  
Adding a new production deployment should not alter development.  
Forcing to provide a separate variable for each deployment that we have just does not sound right.